### PR TITLE
Add Brittany GenOuest instance

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: server.geojson [hunk 1]
```diff
@@ -43,6 +43,19 @@
...
         "type": "Feature",
         "geometry": {
+          "type": "Point",
+          "coordinates": [48.11553,	-1.63883]
+        },
+        "properties": {
+          "Name": "GenOuest Galaxy Server",
+          "URL": "http://galaxy.genouest.org",
+          "Description": "",
+          "Training": "We offer several training course by year, mainly introductive ones.",
```
Comment (bebatut) [inline server.geojson:53]: As you are a Training Instance, can you add: 

```
"marker-size": "medium",
"marker-symbol": "college",
"marker-color": "#bbbbbb"
```

Thanks

**Review by bebatut — COMMENTED** [review]
Thanks

File: server.geojson (hunk 1 above)
Comment (yvanlebras) [reply server.geojson:53]: For sure, thanks!

**Comment by bgruening** [comment]
Thanks @yvanlebras!

**Comment by bgruening** [comment]
@yvanlebras the coordinate was wrong: https://github.com/bgruening/galaxy-maps/blob/master/server.geojson

Most probably, just change the both numbers.

**Pull request merged**